src/zmq/zmqpublishnotifier.cpp

```diff
@@ -138,6 +140,30 @@ bool CZMQPublishHashTransactionNotifier::NotifyTransaction(const CTransaction &t
     return rc == 0;
 }
 
+bool CZMQPublishOwnedHashTransactionNotifier::NotifyTransaction(const CTransaction &transaction)
+{
+    int i, numOuts = transaction.vout.size();
+    bool owned = false;
+    for(i=0; i<numOuts; i++){
```

Comment:
`for (int i =0; i < numOuts; i++) {`

This isn't C, and you already redefine `i` inline below.
Also, spacing.